scripts/check-git-status.sh

```diff
@@ -11,6 +11,6 @@ if [ -z "$(git status --porcelain)" ]; then
   exit 0
 else
   echo "Detected changes in the working directory:"
-  git --no-pager diff --stat HEAD
+  git --no-pager diff HEAD
```

Comment:
This was helpful in debugging the issue. We now emit a full diff when we detect changes instead of just showing _which_ files changed.